components/home/resource-list.tsx

```
@@ -0,0 +1,101 @@
+import React from "react";
+import { RESOURCE_TYPES, ResourceTypeEnum } from "@/lib/constants";
```

ESLint

[error] 'RESOURCE_TYPES' is defined but never used.

(@typescript-eslint/no-unused-vars)